build/postgres-gis-ha/Dockerfile

```diff
@@ -18,92 +28,73 @@ LABEL name="postgres-gis-ha" \
-		--setopt=skip_missing_names_on_install=False \
-		--enablerepo="powertools" \
-		libRmath \
-		pgrouting${POSTGIS_LBL}_${PG_MAJOR//.} \
-		plr${PG_MAJOR//.} \
-		postgis${POSTGIS_LBL}_${PG_MAJOR//.} \
-		postgis${POSTGIS_LBL}_${PG_MAJOR//.}-client \
-		postgresql${PG_MAJOR//.}-plperl \
-		perl \
-		R-core \
+		python3-pip \
+		python3-psutil \
+		python3-psycopg2 \
 	&& ${PACKAGER} -y clean all ; \
 fi
 
 RUN if [ "$BASEOS" = "rhel7" ] ; then \
 	${PACKAGER} -y install \
 		--enablerepo="epel,rhel-7-server-optional-rpms" \
 		--setopt=skip_missing_names_on_install=False \
-		libRmath \
-		pgrouting${POSTGIS_LBL}_${PG_MAJOR//.} \
-		plr${PG_MAJOR//.} \
-		postgis${POSTGIS_LBL}_${PG_MAJOR//.} \
-		postgis${POSTGIS_LBL}_${PG_MAJOR//.}-client \
-		postgresql${PG_MAJOR//.}-plperl \
-		perl \
-		R-core \
-		texinfo-tex \
-		texlive-epsf \
+		python3-pip \
+		python3-psutil \
+		python3-psycopg2 \
 	&& ${PACKAGER} -y clean all --enablerepo="epel,rhel-7-server-optional-rpms" ; \
 fi
 
 RUN if [ "$BASEOS" = "ubi7" ] ; then \
 	${PACKAGER} -y install \
 		--enablerepo="epel,rhel-7-server-optional-rpms" \
 		--setopt=skip_missing_names_on_install=False \
-		libRmath \
-		pgrouting${POSTGIS_LBL}_${PG_MAJOR//.} \
-		plr${PG_MAJOR//.} \
-		postgis${POSTGIS_LBL}_${PG_MAJOR//.} \
-		postgis${POSTGIS_LBL}_${PG_MAJOR//.}-client \
-		postgresql${PG_MAJOR//.}-plperl \
-		perl \
-		R-core \
-		texinfo-tex \
-		texlive-epsf \
+		python3-pip \
+		python3-psutil \
+		python3-psycopg2 \
 	&& ${PACKAGER} -y clean all --enablerepo="epel,rhel-7-server-optional-rpms" ; \
 fi
 
 RUN if [ "$BASEOS" = "ubi8" ] ; then \
-	${PACKAGER} -y --enablerepo="epel,codeready-builder-for-rhel-8-x86_64-rpms" install libaec libdap armadillo \
-	&& ${PACKAGER} -y install \
+	${PACKAGER} -y install \
 		--enablerepo="epel" \
 		--setopt=skip_missing_names_on_install=False \
-		libRmath \
-		pgrouting${POSTGIS_LBL}_${PG_MAJOR//.} \
-		plr${PG_MAJOR//.} \
-		postgis${POSTGIS_LBL}_${PG_MAJOR//.} \
-		postgis${POSTGIS_LBL}_${PG_MAJOR//.}-client \
-		postgresql${PG_MAJOR//.}-plperl \
-		perl \
-		R-core \
-		texlive-epsf \
-	&& ${PACKAGER} -y clean all --enablerepo="epel,codeready-builder-for-rhel-8-x86_64-rpms" ; \
+		python3-pip \
+		python3-psutil \
+		python3-psycopg2 \
+	&& ${PACKAGER} -y clean all --enablerepo="epel" ; \
 fi
 
+# install patroni for Kube
+RUN pip3 install --upgrade python-dateutil \
+	&& pip3 install patroni[kubernetes]=="${PATRONI_VER}"
+
+RUN useradd crunchyadm -g 0 -u 17
+
+ENV PATH="${PGROOT}/bin:${PATH}"
+
+RUN mkdir -p /crunchyadm /tablespaces
+
+# Adjust ownership for the folders to be the "postgres" user and allow the group
+# permissions to match the user ones EXCEPT for the /tablespaces folder, which
+# will only have permissions on the user
+RUN chown -R postgres:postgres /crunchyadm /tablespaces && \
+	chmod -R g=u /crunchyadm /tablespaces
+
 # open up the postgres port
 EXPOSE 5432
 
-ADD bin/postgres-gis-ha /opt/crunchy/bin/postgres-ha
```

Comment:
I think we still need to add bin/postgres-gis-ha after we add the bin/postgres-ha.  Otherwise it doesn't create the postgis extensions, right?

Reply:
Oops you're right -- good catch.